R/call_graph.R

```diff
@@ -39,9 +39,11 @@ call_graph_endpoint <- function(token, operation, ..., options=list(),
 call_graph_url <- function(token, url, ..., body=NULL, encode="json",
                            http_verb=c("GET", "DELETE", "PUT", "POST", "HEAD", "PATCH"),
                            http_status_handler=c("stop", "warn", "message", "pass"),
-                           auto_refresh=TRUE)
+                           auto_refresh=TRUE,
+                           additionalHeaders=NULL, contentType="application/json")
```

Comment:
Rather than extending this function to handle a non-Graph endpoint URL, would be better just to call httr::PUT directly to do the upload

Reply:
sorry it has taken so long to reply.  I wanted to use built in functions because of the token logic.  I would have to recode adding the header, but i'm willing to adjust if based on your preference for this library

Reply:
`call_graph_url` basically just adds the Authorization: header, and makes sure the request and response bodies are handled correctly. Is the Auth header needed here? It doesn't seem to be mentioned in the documentation.

Reply:
Yes, i understand that... But it is doing the work already, I just had to tweak it to expose it, and allow additional headers.  I'm good with making the change to use httr::GET or to keep the code to what i've done already... You are the maintainer, so i'm willing to change to what you recommend.  Just let me know

Reply:
Yes, please change this to call httr::GET directly. I would expect the actual amount of code required to be about the same, if not less.